include/ftof.h

```diff
@@ -0,0 +1,28 @@
+/*! \file ftof.h
+ *  \brief Main header file for Fast/Fenics interface
+ */
+
+
+#ifndef __FTOF_H
+#define __FTOF_H
+
+#include <dolfin.h>
+
+extern "C"
+{
+#include "fasp.h"
+#include "fasp_functs.h"
+    INT fasp_solver_bdcsr_krylov_block_3(block_dCSRmat *A,
+                                       dvector *b,
+                                       dvector *x,
+                                       itsolver_param *itparam,
+                                       AMG_param *amgparam,
+                                       dCSRmat *A_diag);
+#define FASP_BSR     ON  /** use BSR format in fasp */
```

Comment:
I added a PR (#4)[https://github.com/thepnpsolver/modular-pnp/pull/4] off of this branch.  If you like the changes, you can merge it here and then make changes to the naming conventions as you mentioned, and add the ``free()``.  Otherwise, you can just copy over what you like.  Let me know what you think so that we can get *this* PR merged ASAP.